src/kirby/components/list/list.component.html

```diff
@@ -5,7 +5,7 @@
           [class.has-header]="listHeaderTemplate"
           [class.has-footer]="listFooterTemplate"
           (window:resize)="onResize()">
-  <ion-list-header *ngIf="listHeaderTemplate">
+  <ion-list-header *ngIf="listHeaderTemplate" class="list-header">
```

Comment:
Just use `ion-list-header` as selector:
```suggestion
  <ion-list-header *ngIf="listHeaderTemplate">
```

Reply:
Done.